sufia-models/lib/sufia/models/utils.rb

```diff
@@ -1,12 +1,26 @@
+# This module includes a utility methods
 module Sufia
   module Utils
     extend ActiveSupport::Concern
 
+    # retry the block if the conditional call is true unless we hit the maximum tries
+    #
+    # @param number_of_tries [Fixnum] maximum number of times to retry the block
+    # @param condition [Boolean] conditional to call and see if we SHOULD retry
```

Comment:
@mjgiarlo  If you have something to say about it I can add it.  Otherwise it is a ruby concept people should know. Looking at the examples from the contributing doc, the one that passed a block did not comment on it.  In addition the getting started had no blocks.